File: source/code/csharp/account_management/search_dir_accounts_for_create_after_date_req.cs
var accountsCreatedAfterMidnightJan5 = await myDirectory
    .GetAccounts()
    // Jan 5 2016, midnight GMT
    .Where(acct => acct.CreatedAt > new DateTimeOffset(2016, 1, 5, 00, 00, 00, TimeSpan.Zero))
    .ToListAsync();

File: source/code/vbnet/account_management/search_dir_accounts_for_create_after_date_req.vb
Dim accountsCreatedAfterMidnightJan5 = Await myDirectory _
    .GetAccounts() _
    .Where(Function(acct) acct.CreatedAt > New DateTimeOffset(2016, 1, 5, 0, 0, 0, TimeSpan.Zero)) _
    .ToListAsync() ' Jan 5 2016, midnight GMT
